libs/ez-api/ezAPI.ml

```diff
@@ -398,6 +398,20 @@ let section_name s = s.section_name
 
 let nservices () = !nservices
 let id s = s.doc.doc_id
+
+let services_map f = List.map f @@ List.rev !services
```

Comment:
I would prefer `List.rev_map`, which is both tail-recursive, and does exactly what you want